packages/pwa/app/components/link/index.jsx

```diff
@@ -11,23 +11,43 @@ import {Link as SPALink, NavLink as NavSPALink} from 'react-router-dom'
 import {buildPathWithUrlConfig} from '../../utils/url'
 import useSite from '../../hooks/use-site'
 import useLocale from '../../hooks/use-locale'
+import {matchRoute} from '../../utils/routes-utils'
+import routes from '../../routes'
 
 const Link = React.forwardRef(({href, to, useNavLink = false, ...props}, ref) => {
```

Comment:
Chatted with Adam, we are looking at moving this into the SDK.

Reply:
I wonder whether the "link out" for paths that aren't SPA routable should be the default behaviour or opt in via a prop.